res/values/strings.xml

```diff
@@ -346,8 +346,8 @@ there are any).</string>
     <!-- SBB is german/international, CFF is french and FFS is italian! So this IS translatable. -->
     <string name="np_name_sbb" translatable="true">SBB</string>
     <string name="np_desc_sbb">Switzerland</string>
-    <string name="np_desc_vbl">Luzern</string>
-    <string name="np_desc_zvv">Zurich</string>
+    <string name="np_desc_vbl">Lucerne</string>
+    <string name="np_desc_zvv">Zürich</string>
```

Comment:
Thanks! However, I still prefer to leave it as "Zurich" as this is also correct and the main reason: Every time you change a source string *all* translations for this string have to be redone. I don't think this change justifies creating unnecessary work for all translators.